cirq-core/cirq/sim/clifford/stabilizer_state_ch_form.py

```diff
@@ -386,7 +389,7 @@ def apply_global_phase(self, coefficient: value.Scalar):
         self.omega *= coefficient
 
     def measure(
-        self, axes: Sequence[int], seed: 'cirq.RANDOM_STATE_OR_SEED_LIKE' = None
+        self, axes: Sequence[int], seed: Optional['cirq.RANDOM_STATE_OR_SEED_LIKE'] = None
```

Comment:
Please, keep the seed type as it was, RANDOM_STATE_OR_SEED_LIKE is typed as Any so None should be fine there.  Add a `parse_random_state()` call so that `self._measure()` gets a correct type.

Reply:
The seed type is assigned to `None` so it's still `Optional`. This is being explicit even though we haven't turned on [no-implicit-optional](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-optional).

Reply:
I haven't confirmed that 

```python
VarType = Any
def hi(var: VatType=None)
```
Fails with `no-implicit-optional` though.